### PR TITLE
fix(tooltiptable): fix non-unique react keys

### PR DESCRIPTION
**Pull request by masoudmanson**
## Summary

**TooltipTable**
Shortcut ticket: #356

### CODE REVIEW

File: src/core/TooltipTable/index.tsx [hunk 1]
```diff
@@ -22,15 +22,18 @@ const TooltipTableContent = (props: TooltipTableContentProps): JSX.Element => {
   return (
     <TableContainer {...rest}>
       {contentAlert && <Alert>{contentAlert}</Alert>}
-      {data?.map((section) => (
-        <Section disabled={section.disabled} key={`${section.label}`}>
+      {data?.map((section, index) => (
+        <Section
+          disabled={section.disabled}
+          key={`${section.label?.concat(index.toString())}`}
```
Comment (tihuan) [inline src/core/TooltipTable/index.tsx:28]: super nit: Maybe this? Personally seems more readable 😄 
```suggestion
          key={section.label + String(index)}
```

File: src/core/TooltipTable/index.tsx [hunk 2]
```diff
@@ -22,15 +22,18 @@ const TooltipTableContent = (props: TooltipTableContentProps): JSX.Element => {
...
+        >
           <SectionLabel disabled={section.disabled} label={section.label}>
             {section.label}
           </SectionLabel>
           <Table size="small">
             <TableBody>
-              {section.dataRows.map((row) => (
-                <TableRow key={row.label}>
+              {section.dataRows.map((row, rowIndex) => (
+                <TableRow key={row.label.concat(rowIndex.toString())}>
```
Comment (tihuan) [inline src/core/TooltipTable/index.tsx:36]:
```suggestion
                <TableRow key={row.label + String(rowIndex)}>
```

**Review by tihuan — APPROVED** [review]
Looks v good! Thanks for the quick fix, Masoud 🙏 One tiny comment, otherwise LGTM 🚀

File: src/core/TooltipTable/index.tsx (hunk 1 above)
Comment (tihuan) [reply src/core/TooltipTable/index.tsx:28]: Also prefer `String(foo)` over `foo.toString()`, so when `foo` is `undefined` or `null`, it won't throw a runtime TypeError and will just render string `undefined ` and `null` instead!

Comment (masoudmanson) [reply src/core/TooltipTable/index.tsx]: Woow ,Thanks for the suggestion. I just refactored it 🥇 
Much better now :D

**Pull request merged**